Timestamp format is also different, shouldn't be an issue other than for testing:

```
AssertionError: '2010-10-06T21:46:50' != '2010-10-06 21:46:50'
- 2010-10-06T21:46:50
?           ^
+ 2010-10-06 21:46:50
?           ^

```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -8,7 +8,7 @@ workflows:
           filters:
             branches:
               ignore: master
-      - test--python-2:
+      - test-python-2:
           filters:
             branches:
               ignore: master

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -1,0 +1,98 @@
+version: 2.1
+
+workflows:
+  version: 2
+  test:
+    jobs:
+      - test-python-3:
+          filters:
+            branches:
+              ignore: master
+      - test--python-2:
+          filters:
+            branches:
+              ignore: master
+
+waitforapt: &waitforapt
+  name: Remove cloud init lock
+  command: |
+      while [ ! -f /var/lib/cloud/instance/boot-finished ]; do echo 'Waiting for cloud-init...'; sleep 10; done
+      while sudo fuser /var/lib/dpkg/lock >/dev/null 2>&1; do echo 'Waiting for autoupdates to complete...'; sleep 10; done
+      echo 'Waiting for instance to really be ready...'
+      sleep 30
+      sudo rm -rf /var/lib/apt/lists/lock
+      sudo rm /var/lib/dpkg/lock && sudo dpkg --configure -a
+
+
+install_cognitiveatlas: &install_cognitiveatlas
+  name: install cognitiveatlas
+  command: |
+     $HOME/conda/bin/pip uninstall cognitiveatlas --yes || echo "Not installed"
+     $HOME/conda/bin/python setup.py install
+
+install_python_3: &install_python_3
+  name: install Python 3.5 dependencies
+  command: | 
+      ls $HOME
+      if [ ! -d "/home/circleci/conda" ]; then
+          wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+          /bin/bash Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/conda
+          export PATH=$HOME/conda/bin:$PATH
+          $HOME/conda/bin/python setup.py install
+       else
+           echo "Miniconda 3 is already installed, continuing to build."
+       fi
+
+install_python_2: &install_python_2
+  name: install Python 3.5 dependencies
+  command: | 
+      ls $HOME
+      if [ ! -d "/home/circleci/conda" ]; then
+          wget https://repo.anaconda.com/miniconda/Miniconda2-latest-Linux-x86_64.sh
+          /bin/bash Miniconda2-latest-Linux-x86_64.sh -b -p $HOME/conda
+          export PATH=$HOME/conda/bin:$PATH
+          $HOME/conda/bin/python setup.py install
+       else
+           echo "Miniconda 2 is already installed, continuing to build."
+       fi
+
+test_cognitiveatlas: &test_cognitiveatlas
+  name: Test Cognitive Atlas (Used for Python 2 and 3)
+  command: |
+        cd ~/repo/cognitiveatlas
+        $HOME/conda/bin/python -m unittest tests.test_api
+
+jobs:
+  test-python-3:
+    machine: true
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v1-dependencies
+      - run: *install_python_3
+      - run: *waitforapt
+      - run: *install_cognitiveatlas
+      - save_cache:
+          paths:
+            - /home/circleci/conda
+          key: v1-dependencies        
+      - run: *test_cognitiveatlas
+
+  test-python-2:
+    machine: true
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v1-dependencies
+      - run: *install_python_2
+      - run: *waitforapt
+      - run: *install_cognitiveatlas
+      - save_cache:
+          paths:
+            - /home/circleci/conda
+          key: v1-dependencies        
+      - run: *test_cognitiveatlas

--- a/README.md
+++ b/README.md
@@ -9,8 +9,15 @@ This API gives python access to the RESTful API in the Cognitive Atlas (alpha).
 - disorders
 - contrasts
 
-
 ### Documentation
 
 Read our complete [documentation](https://cogat-python.readthedocs.org) for getting started.
 
+
+### Python 2.* notice
+
+If you need cognitiveatlas specifically for python 2, the last released version was 0.0.18:
+
+```bash
+$ pip install cognitiveatlas==0.0.18
+```

--- a/cognitiveatlas/tests/test_api.py
+++ b/cognitiveatlas/tests/test_api.py
@@ -2,87 +2,94 @@
 
 """
 Test core functions of Cognitive Atlas python API wrapper
-
 """
 
-from numpy.testing import assert_array_equal, assert_almost_equal, assert_equal
-from cognitiveatlas.api import get_concept, search, get_task, get_disorder
-from nose.tools import assert_true, assert_false
+import unittest
+from cognitiveatlas.api import ( 
+    get_concept, 
+    search,
+    get_task, 
+    get_disorder
+)
 
-'''Test concepts queries'''
-def test_concepts():
-    print("### TESTING CONCEPT QUERIES:")
-    concept_id = "trm_5022ef7599294"
-    concept_name = "anxiety"
-    contrast_id = "cnt_5299143fed521"
+class TestAPI(unittest.TestCase):
 
-    # concept_id
-    result = get_concept(id=concept_id)
-    assert_equal(result.json[0]["name"],concept_name)
+    def test_concepts(self):
+        '''Test concepts queries'''
+        print("### TESTING CONCEPT QUERIES:")
+        concept_id = "trm_5022ef7599294"
+        concept_name = "anxiety"
+        contrast_id = "cnt_5299143fed521"
 
-    # concept_name
-    result = get_concept(name=concept_name)
-    assert_equal(result.json[0]["id"],concept_id)
+        # concept_id
+        result = get_concept(id=concept_id)
+        self.assertEqual(result.json["name"],concept_name)
 
-    # contrast_id
-    result = get_concept(contrast_id=contrast_id)
-    assert_true(any(concept_id==obj["id"] for obj in result.json))
-    #assert_equal(result.json[0]["id"],concept_id)
+        # concept_name
+        result = get_concept(name=concept_name)
+        self.assertEqual(result.json["id"],concept_id)
 
-    # concept_id and concept_name
-    result = get_concept(id=concept_id,name=concept_name)
-    assert_equal(result.json[0]["name"],concept_name)
+        # contrast_id
+        result = get_concept(contrast_id=contrast_id)
+        self.assertTrue(any(concept_id==obj["id"] for obj in result.json))
+       
+        # concept_id and concept_name
+        result = get_concept(id=concept_id,name=concept_name)
+        self.assertEqual(result.json["name"],concept_name)
 
-    # concept_id, and contrast_id
-    result = get_concept(id=concept_id,contrast_id=contrast_id)
-    assert_equal(result.json[0]["name"],concept_name)
+        # concept_id, and contrast_id
+        result = get_concept(id=concept_id,contrast_id=contrast_id)
+        self.assertEqual(result.json["name"],concept_name)
 
-    # concept_name and contrast_id
-    result = get_concept(name=concept_name,contrast_id=contrast_id)
-    assert_equal(result.json[0]["id"],concept_id)
-    
+        # concept_name and contrast_id
+        result = get_concept(name=concept_name,contrast_id=contrast_id)
+        self.assertEqual(result.json["id"],concept_id)
 
-'''Test search query'''
-def search(query):
-    result = search(query="anxiety")
-    assert_true(len(result.json)>20)
+    def test_search(self):
+        '''Test search query'''
+        result = search(query="anxiety")
+        self.assertTrue(len(result.json)>20)
 
-'''Test task queries'''
-def test_task():
-    print("### TESTING TASK QUERIES:")
-    task_id = "trm_4cacee4a1d875"
-    task_name = "mixed gambles task"
+    def test_task(self):
+        '''Test task queries'''
 
-    # task_id and task_name
-    result = get_task(id=task_id,name=task_name)
-    assert_equal(result.json[0]["type"],"task")
-    assert_equal(result.json[0]["event_stamp"],"2010-10-06 21:46:50")
+        print("### TESTING TASK QUERIES:")
+        task_id = "trm_4cacee4a1d875"
+        task_name = "mixed gambles task"
 
-    # task_id
-    result = get_task(id=task_id)
-    assert_equal(result.json[0]["name"],task_name)
+        # task_id and task_name
+        result = get_task(id=task_id,name=task_name)
+        self.assertEqual(result.json["type"],"task")
+
+        # task_id
+        result = get_task(id=task_id)
+        self.assertEqual(result.json["name"],task_name)
  
-    # task_name
-    result = get_task(name=task_name)
-    assert_equal(result.json[0]["id"],task_id)
+        # task_name
+        result = get_task(name=task_name)
+        self.assertEqual(result.json["id"],task_id)
 
+    def test_disorder(self):
+        '''Test disorder queries'''
 
-'''Test disorder queries'''
-def test_disorder():
-    print("### TESTING DISORDER QUERIES:")
-    disorder_id = "dso_3324"
-    disorder_name = "mood disorder"
+        print("### TESTING DISORDER QUERIES:")
+        disorder_id = "dso_3324"
+        disorder_name = "mood disorder"
 
-    # disorder_id and disorder_name
-    result = get_disorder(id=disorder_id,name=disorder_name)
-    assert_equal(result.json[0]["name"],disorder_name)
-    assert_equal(result.json[0]["is_a_fulltext"],"cognitive disorder")
-    assert_equal(result.json[0]["event_stamp"],"2013-11-20 15:38:27")
+        # disorder_id and disorder_name
+        result = get_disorder(id=disorder_id,name=disorder_name)
+        print(result)
+        self.assertEqual(result.json["name"],disorder_name)
+        print(result.json)
+        self.assertEqual(result.json["type"],"disorder")
 
-    # disorder_id
-    result = get_disorder(id=disorder_id)
-    assert_equal(result.json[0]["name"],disorder_name)
+        # disorder_id
+        result = get_disorder(id=disorder_id)
+        self.assertEqual(result.json["name"],disorder_name)
  
-    # disorder_name
-    result = get_disorder(name=disorder_name)
-    assert_equal(result.json[0]["id"],disorder_id)
+        # disorder_name
+        result = get_disorder(name=disorder_name)
+        self.assertEqual(result.json["id"],disorder_id)
+
+if __name__ == '__main__':
+    unittest.main()

